db/migrate/20200618153503_create_messages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMessages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :messages do |t|
+      t.string :title
+      t.integer :content
```

Comment:
型を間違えています。contentは文章ですので`string`ですね。

```suggestion
      t.string :content
```

修正後に， 次を実行すれば修正できます。

```
rails db:migrate:reset
```